### DISCUSSION TITLE
Is it possible to display personalized item-bar for simple properties?

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#847] [Q&A]
Hi

Mavo allows to display an `mv-item-bar mv-ui `on collections to delete, add or move an item (great)

I would like to know if there's a way to display a personalized bar on a **simple** property ?

In this [codepen](https://codepen.io/galinhalx/pen/wvyGeog) , I've been able to make it works with collections, 
(this is just a example about how it should look like)
but I would like to know if mavo allows to do the same in a simple property.

In my usecase, I want to add a `<button>` to this personalized bar, who will do an action (toggle a class on the item)

**Comment by LeaVerou** [comment]
Oh that's an interesting idea. We haven't considered it before. Let me think how we could support this in a sufficiently generic way.
